src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int maxRandomNumber = 100;
```

Comment:
is it a constant? - https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
constant should contain two keywords: static and final
and the name should be written in upper case